packages/editor/src/components/blocks/select/Select.tsx

```diff
@@ -40,14 +40,14 @@ export const SelectComponent: ComponentConfig<SelectProps> = {
     dynamicItemsLabel: {
       subsection: 'Dynamic Options',
       label: 'Dynamic Label',
```

Comment:
maybe object label is better

Reply:
like this?
![grafik](https://github.com/user-attachments/assets/b41ae615-55b0-4977-af5c-0ea35a7166c5)

Reply:
and the value too